oak/server/status.h

```diff
@@ -17,6 +17,8 @@
 #ifndef OAK_SERVER_STATUS_H_
 #define OAK_SERVER_STATUS_H_
 
+#include <stdint.h>
```

Comment:
Actually, this might be better as `<cstdint>` as we're in C++-land.

Reply:
I've just had a big fight with stdint and cstdint :) For the module code with clang, as I am compiling in freestanding mode and lack stdlib, cstdint won't compile as it needs support for threading.... FUN! 

In this case cstdint might be better (as it's not polluting the namespace), but then we either need a using namespace std; or std:: (assuming the implementation does what is supposed to do :) )

My head hurts :(